aspnetcore/host-and-deploy/linux-nginx.md

```diff
@@ -143,6 +168,21 @@ With the preceding configuration file and default server, Nginx accepts public t
 
 Once the Nginx configuration is established, run `sudo nginx -t` to verify the syntax of the configuration files. If the configuration file test is successful, force Nginx to pick up the changes by running `sudo nginx -s reload`.
 
+To directly run the app on the server:
+
+1. Navigate to the app's directory.
+1. Run the app's executable: `./<app_executable>`.
+
+If a permissions error occurs, change the permissions:
+
+```console
+chmod u+x <app_executable>
+```
```

Comment:
Most likely the nginx user isn't running as the current running user. `chmod u+x` wouldn't really help solve the problem. I'm not sure if `chmod +x` has any security implications? @blowdart

Reply:
I have no idea, it's linux :(

Reply:
Not sure what to do on this one @shirhatti.

Reply:
Leave it as `u+x`. Less permissions is better, though it might not fix the issue being called out.